content/en/docs/languages/java/resources.md

```diff
@@ -100,3 +100,10 @@ SdkLoggerProvider sdkLoggerProvider = SdkLoggerProvider.builder()
     ...
     .build();
 ```
+
+## Next steps
+
+There are more resource detectors that you can add to your configuration. You
+can find them in the
+[opentelemetry-java-contrib](https://github.com/open-telemetry/opentelemetry-java-contrib)
+repository.
```

Comment:
Could perhaps also add a note that most of those are in the [resource-providers](https://github.com/open-telemetry/opentelemetry-java-contrib/tree/main/resource-providers) module...I think that could be some additional steering.

Reply:
Ah, good catch. The vendor specific resource detection modules like [aws-resources](https://github.com/open-telemetry/opentelemetry-java-contrib/tree/main/aws-resources) and [gcp-resources](https://github.com/open-telemetry/opentelemetry-java-contrib/tree/main/gcp-resources) are listed directly under the root repository, while the common ones are bundled under `resource-providers`. 

I will raise this question in the next SIG meeting to club them under a single folder for better discoverability.
 
However, given the current directory structure, do you think we should mention to specifically look for `resource-providers` module ? I think that might lead the users to think that it is the only one in the repo.

Reply:
Had a discussion in the Java SIG meeting regarding this, it seems like it would be better to provide a list of locations where to find resource-detectors. I will update the PR to do so.